CHANGELOG.md

```diff
@@ -5,13 +5,47 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.1.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.1.0) - 2023-07-18
 
-## [Zebra 1.0.1](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.1) - 2023-XX-XX
+Zebra now uses less memory after the final checkpoint by deduplicating note commitment trees in the non-finalized state.
```

Comment:
Optional: Is this a noticeable amount of memory?

I think we're talking about less than a kilobyte per note commitment tree, for 3 shielded protocols, for 100 blocks (so less than 1MB). It's also possible we were already de-duplicating the trees by cloning the `Arc` rather than the whole tree.

Reply:
> Is this a noticeable amount of memory?

I don't think so, but I wasn't sure.

> It's also possible we were already de-duplicating the trees by cloning the Arc rather than the whole tree.

That's correct, except if/when updating trees with new note commitments can reproduce the same tree.

Reply:
> > It's also possible we were already de-duplicating the trees by cloning the Arc rather than the whole tree.
> 
> That's correct, unless updating trees with new note commitments can reproduce the same tree.

No, it's not possible to add a commitment and get the same tree. Adding a new note commitment always changes the tree, because it always adds a leaf, and sometimes removes a subtree.